lib/Driver/ToolChains.cpp

```diff
@@ -1781,6 +1803,14 @@ bool toolchains::Android::shouldProvideRPathToLinker() const {
   return false;
 }
 
+std::string toolchains::Fuchsia::getDefaultLinker() const {
+  return "lld";
```

Comment:
This should be `"ld.lld"`.

Reply:
Fix in flight. I thought "-fuse-ld=ld.lld" looked funnier than "-fuse-ld=lld" that swift invokes on clang.

Reply:
Ah, if this is what Swift passes to `-fuse-ld=` then `"lld"` is correct, if this is the name of the binary that Swift will invoke then it should be `"ld.lld"` (I haven't looked at how this is wired up internally).

Reply:
yeah it's for [-fuse-lld](https://github.com/google/swift/blob/0c253b7fce499b31a312d2912e1c0a20ca62b360/lib/Driver/ToolChains.cpp#L1594)